src/data/UserD2Repository.ts

```diff
@@ -64,59 +65,35 @@ export class UserD2Repository implements UserRepository {
             .value();
     }
 
-    async getByUserName(username: string): Async<User | undefined> {
+    async updateUserName(
+        fromProperty: string,
+        toProperty: string,
+        usersToUpdate: UserMigrate[]
+    ): Async<MigrationResult> {
```

Comment:
This method does not seem to do anything with the username, its now more generic, can be renamed.